server/README.md

```diff
@@ -0,0 +1,141 @@
+# Language server bitbkae
+
+Language server for bitbake.
+
+![screenshot](https://github.com/yoctoproject/vscode-bitbake/assets/32936898/82e16428-e5fb-4e37-a2fa-b56c00cc4f4c)
+
+## Install
+
+```sh
+npm install -g language-server-bitbake
+```
+
+## Configure
+
+In the following instructions:
+
+- For windows, replace `~/.config` to `~/AppData/Local`
+- For macOS, replace `~/.config` to `~/Library`
+
+### (Neo)[Vim](https://www.vim.org)
```

Comment:
Instructions are quite clear, I could get it to work!
The only missing step was to install the bitbake language plugin. I added `Plug 'kergoth/vim-bitbake'` in `init.vim` 
![image](https://github.com/yoctoproject/vscode-bitbake/assets/19970889/0e11b570-a035-4828-91a6-ba35841268fd)

Reply:
I was running 0.6.1. That explains it. thanks